tools/provision/lib.sh

```diff
@@ -188,6 +188,34 @@ function install_yara() {
   fi
 }
 
+function install_openssl() {
+  SOURCE=openssl-1.0.2d
+  TARBALL=$SOURCE.tar.gz
+  URL=http://openssl.org/source/$TARBALL
+
+  if provision openssl /usr/lib/libssl.so.1.0.0; then
+    pushd $SOURCE
+    CC="$CC" CXX="$CXX" ./config --prefix=/usr --openssldir=/etc/ssl --libdir=lib shared zlib-dynamic enable-shared
```

Comment:
nope, no specific reason. Will try to build with /usr/local . Thanks for pointing out